### PR TITLE
Update quality.yml

### PR DESCRIPTION
**Pull request by eerison**

### CODE REVIEW

**Comment by eerison** [comment]
Hey @Willian1661 could you review please 

The pipeline wasn't running for master branch

File: .github/workflows/quality.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ name: quality
 
 on: 
   push:
-    branches: [ $default-branch ]
+    branches: [master]
```
Comment (Willian1661) [inline .github/workflows/quality.yml:5]: i think it's => 
  branches: 
    - 'master'
 
i saw some examples that use this format => branches: [master] so i don't know the difference, but try it.

Comment (eerison) [reply .github/workflows/quality.yml:5]: It works with

- or []

Let's use with [], to follow pull request example

Comment (eerison) [reply .github/workflows/quality.yml:5]: If everything is fine just merge it

Comment (Willian1661) [reply .github/workflows/quality.yml:5]: for me it's all right, all the pushes that we'll do it'll be into the branch master.
so for every push event it'll trigger this workflow and the same with the pull_request.
so i believe that's all right.

**Review by Willian1661 — COMMENTED** [review]
all the pushes that we'll do it'll be into the branch master.
so for every push event it'll trigger this workflow and the same with the pull_request.
so i believe that's all right.

**Pull request merged**